spec.html

```diff
@@ -5069,9 +5069,9 @@ <h1>SameValueNonNumeric ( _x_, _y_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-abstract-relational-comparison" aoid="Abstract Relational Comparison">
-      <h1>Abstract Relational Comparison</h1>
-      <p>The comparison _x_ &lt; _y_, where _x_ and _y_ are values, produces *true*, *false*, or *undefined* (which indicates that at least one operand is *NaN*). In addition to _x_ and _y_ the algorithm takes a Boolean flag named _LeftFirst_ as a parameter. The flag is used to control the order in which operations with potentially visible side-effects are performed upon _x_ and _y_. It is necessary because ECMAScript specifies left to right evaluation of expressions. The default value of _LeftFirst_ is *true* and indicates that the _x_ parameter corresponds to an expression that occurs to the left of the _y_ parameter's corresponding expression. If _LeftFirst_ is *false*, the reverse is the case and operations must be performed upon _y_ before _x_. Such a comparison is performed as follows:</p>
+    <emu-clause id="sec-abstract-relational-comparison" aoid="AbstractRelationalComparison">
+      <h1>AbstractRelationalComparison ( _x_, _y_ [ , _LeftFirst_ ] )</h1>
```

Comment:
Thoughts on calling this `AbstractLessThanComparison`, just so it's clear what the order is? (Or some other such name.)

Reply:
When this came up in IRC, there was some sentiment for keeping the names the same so as not to invalidate books and such. But if that's off the table, then yes, I think we can come up with better names.

> Thoughts on calling this `AbstractLessThanComparison`, just so it's clear what the order is?

That would be an improvement. I think "LessThan" implies "Comparison", so maybe just `AbstractLessThan`. Further, `Abstract` doesn't seem right: abstract as opposed to what? So maybe just `LessThan`. Or maybe `GeneralLessThan` if you want to convey that it accepts any two ECMAScript language values.

For `{Abstract,Strict}EqualityComparison`: again, the "Comparison" seems redundant. And again, "Abstract" doesn't seem right; to contrast with "Strict", I think "Loose" would be better. So maybe `{Loose,Strict}Equality` or `{Loose,Strict}Equals`.

Reply:
I do think there's some value in keeping the existing names. It's just that for this one, in particular, the fact that the calling convention will no longer mean `<` will make it ambiguous in a way that "StrictEqualityComparison" is not.

Still, I'm open to renaming the others as well, I guess.

Reply:
If we do rename them, I vote for `isStrictlyEqual` / `isLooselyEqual` or `isAbstractlyEqual`, or similar (ie, predicate naming)

Reply:
`IsLessThan`, `IsStrictlyEqual`, and `IsRoughlyEqual` would be my preference, but I don't have strong feelings here.

Reply:
I like @ljharb's suggestion of "loosely" better than "roughly", I think, because "roughly" makes me think about floating point epsilons rather than "not strict".

Anyway, those sound ok to me.

Reply:
My preference is `IsLessThan`, `IsStrictlyEqual`, `IsLooselyEqual`.

Reply:
Is "loose" a common term used to refer to `==` comparisons?

Reply:
Well, it's [what MDN uses](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#loose_equality_using), as well as [this popular book](https://exploringjs.com/impatient-js/ch_operators.html#loose-equality-and). And [this book](https://github.com/getify/You-Dont-Know-JS/blob/8861041133f496edce0d03885e2e998d50c3414a/get-started/ch2.md#coercive-comparisons) also says "the == operator, generally referred to as the 'loose equality' operator". So, pretty common, I think.

Reply:
It looks like "loose" is about as common as "strict": googling "javascript loose equality" gets me about 762,000 results, compared to about 790,000 for "javascript strict equality".